semantic-release/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "test",
```

Comment:
if the name is must, please set something more relevant, for example "Semantic release scripts"